README.md

```diff
@@ -1,3 +1,10 @@
 # biotope-generator
 
-A file generator for the biotope project user by the biotope cli.
+A file generator for the biotope project, used by the biotope cli.
+
+## biotope
```

Comment:
Please change "biotope" to "Biotope". Then we are consistent with the way we agreed to write the name of Biotope in general copy texts. Thanks

Reply:
Done